components/filter/index.js

```diff
@@ -20,7 +22,11 @@ const Filter = (props) => {
     >
       <div className="icon">
         <Image
-          src="/assets/filter-icon.svg"
+          src={
+            theme === 'light'
+              ? '/assets/filter-icon.svg'
```

Comment:
Move the image `src` and the content to `constant` file. This logic could be a component as reusable. This is also getting used in `Nav` components also.